contracts/experimental/ERC20ProxyForZRC2/contracts/ScillaConnector.sol

```diff
@@ -6,6 +6,39 @@ library ScillaConnector {
     uint private constant SCILLA_CALL_PRECOMPILE_ADDRESS = 0x5a494c53;
     uint private constant SCILLA_STATE_READ_PRECOMPILE_ADDRESS = 0x5a494c92;
 
+    /**
+     * @dev Calls a ZRC2 contract function with two arguments
+     * @param target The address of the ZRC2 contract
+     * @param tran_name The name of the function to call
+     * @param arg1 The first argument to the function
+     */
+    function callu128(
```

Comment:
FWIW Solidity allows function overloading based on parameter types, so we could still name this `call`?

Reply:
Hmph. It works now - sorry & thanks for the spot! Will update - confusingly, the other PR .. sorry!